### DISCUSSION TITLE
About USB

### DISCUSSION THREAD

**Discussion by mayataotaotao** [Ultrawipf/OpenFFBoard#72] [General]
Can I update the version using customhid instead of tinyusb?

**Comment by Ultrawipf** [comment]
The usb system was previously not using tinyusb and was built on the ST hal usb system. That was very difficult to maintain so we decided to use tinyusb instead as that seems to work very well with freertos and allows easy switching between device configurations. Unless you have a very good reason to change it i would stay with tinyusb.
